### DISCUSSION TITLE
macOS (x64/x86-64; ARM64 (Apple Silicon M1+))

### DISCUSSION THREAD

**Discussion by TigerPhenix** [vlang/v#13830] [Installation Issues]
Please comment with or ask questions about your experience or trouble installing V on Apple's macOS operating system.

IMPORTANT: 32-bit / IA-32 / x86 usage is NOT *officially* supported and would only be available up to Mojave (macOS 10.14). "Hackintosh" systems are also not *officially* supported.

**Comment by TigerPhenix** [comment]
There is some error when make v on my mac (os version 10.15.7), Can anyone help? Thanks very much .
Flowing is the output message :
```
➜  v git:(master) make
cd ./vc && git clean -xf && git pull --quiet
cc  -g -std=gnu11 -w -o v ./vc/v.c  -lm -lpthread
./vc/v.c:1232:2: error: VERROR_MESSAGE Header file <atomic.h>, needed for module
      `sync.stdatomic` was not found. Please install the corresponding
      development headers.
#error VERROR_MESSAGE Header file <atomic.h>, needed for module `sync.st...
 ^
./vc/v.c:5143:2: error: unknown type name 'atomic_uintptr_t'
        atomic_uintptr_t write_adr;
        ^
./vc/v.c:5144:2: error: unknown type name 'atomic_uintptr_t'
        atomic_uintptr_t read_adr;
        ^
./vc/v.c:5145:2: error: unknown type name 'atomic_uintptr_t'
        atomic_uintptr_t adr_read;
        ^
./vc/v.c:5146:2: error: unknown type name 'atomic_uintptr_t'
        atomic_uintptr_t adr_written;
        ^
5 errors generated.
make: *** [all] Error 1
➜  v git:(master)
```

**Reply by harens** [reply]
Adding `-I <V DIRECTORY>/thirdparty/stdatomic/nix/` made some progress for me. This flag should've been added by [this declarations file](https://github.com/vlang/v/blob/master/vlib/sync/stdatomic/1.declarations.c.v) I think, but for some reason it wasn't.

EDIT: It seems like the makefile has been updated since you last run it to include this change.

**Reply by TigerPhenix** [reply]
Thanks, I just compile the project success by doing nothing except delete the old repo and use the latest code from the master branch, and the final output message is ` V has been successfully built. V 0.2.4 2d867a2`
